org.ricsdev.pathnv.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.ricsdev.pathnv
```

Comment:
ricsdev.org goes to an empty landing page https://ricsdev.org/lander How do I know you control the website? Can you put something up about this project?